blocks/minesweeper/src/app.ts

```diff
@@ -0,0 +1,65 @@
+import { BlockElementBase } from "@blockprotocol/graph/custom-element";
+import { getRoots } from "@blockprotocol/graph/stdlib";
+import { PropertyValues } from "lit";
+// eslint-disable-next-line import/extensions
+import { unsafeHTML } from "lit/directives/unsafe-html.js";
+import { mine_sweeper } from "mine-sweeper-tag";
+
+import { BlockEntity } from "./types/generated/block-entity";
+
+const colsKey: keyof BlockEntity["properties"] =
+  "https://blockprotocol-molpob88k.stage.hash.ai/@ciaranm/types/property-type/number-of-columns/";
+const minesKey: keyof BlockEntity["properties"] =
+  "https://blockprotocol-molpob88k.stage.hash.ai/@ciaranm/types/property-type/number-of-mines/";
```

Comment:
Just a reminder to update these

Reply:
https://github.com/hashintel/hash/pull/2222/commits/afca30260994fac67c35cca5eb0fd41f53dc9855